src/lib/factories.js

```diff
@@ -46,7 +46,7 @@ export function createShorthand(Component, mapValueToProps, val, defaultProps =
   } else if (_.isPlainObject(val)) {
     type = 'props'
     usersProps = val
-  } else if (_.isString(val) || _.isNumber(val)) {
+  } else if (_.isString(val) || _.isNumber(val) || _.isArray(val)) {
```

Comment:
I had a sudden moment of clarity thinking about this haha. There are really two separate types of shorthands we're talking about:

### What is shorthand?
A way of describing component using a literal value, a props object, or an complete React.Element

### Item Shorthand
What we've dealt with thus far. Describes an individual component that may or may not have children. If it does have children, the children tend to be heterogeneous/different (e.g. a label can have multiple children but they're all different: icon, image, content, detail, delete-button)

#### Valid propTypes:  
- string/number
- Object (props)
- Element

#### Example Components
- Label
- Icon
- TableCell

### Collection Shorthand
What this PR is introducing. Describes an individual component that has a set of homogenous children (e.g. a menu has 1+ items that are all the same).

#### Valid propTypes:
- array (of "item shorthand"s for set of children)
- Object (props)
- Element

#### Examples
- Menu
- List
- TableRow